modules/structs/user.go

```diff
@@ -93,3 +93,12 @@ type UserSettingsOptions struct {
 	HideEmail    *bool `json:"hide_email"`
 	HideActivity *bool `json:"hide_activity"`
 }
+
+// RenameUserOption options when renaming a user
+type RenameUserOption struct {
+	// New username for this user. This name cannot be in use yet by any other user.
+	//
+	// required: true
+	// unique: true
+	NewName string `json:"new_username" binding:"Required"`
```

Comment:
Why not `new_name`? It looks more consistent, because `name` is used in all other old code.

Reply:
Here I would recommend sticking to the naming used in the API, as that is meant to be read & understood by other parties. That means consistency within the API is (IMHO) more important than consistency with internal code.

Reply:
I see. `CreateUserOption` also use `username`, then it seems fine.